scripts/linters/warranted_angular_security_bypasses.py

```diff
@@ -19,11 +19,11 @@
 from __future__ import absolute_import  # pylint: disable=import-only-modules
 from __future__ import unicode_literals  # isort:skip pylint: disable=import-only-modules
 
-# Contains the tuples of all files that bypasses angular's security mechanisms.
+# Contains the tuples of all files that bypasses angular's SECURITY mechanisms.
```

Comment:
No -- this should still be "security". I was referring to a typo in the variables below; EXCLUDED is also misspelled.

WITHOUT is also now misspelled ...